src/npctalk.cpp

```diff
@@ -1980,11 +1982,18 @@ void talk_effect_fun_t::set_bulk_trade_accept( bool is_trade, bool is_npc )
                     }
                     for( int i = 0; i < buyer_has; i++ ) {
                         seller->i_add( pay );
-                        price -= d.beta->value( pay );
+                        price -= value;
                     }
+                } else {
+                    debugmsg( "%s pays in bulk_trade_accept with faction currency worth 0!",
+                              d.beta->disp_name() );
                 }
-                d.beta->add_debt( price );
+            } else {
+                debugmsg( "%s has no faction currecy to pay with in bulk_trade_accept!",
```

Comment:
```suggestion
                debugmsg( "%s has no faction currency to pay with in bulk_trade_accept!",
```

Reply:
Sheesh, I cannot spell this morning.  Fixed.